lib/router.js

```diff
@@ -0,0 +1,106 @@
+var fs = require("fs");
+
+module.exports = function(req, res) {
+
+  var path = req.url.split("/");
+  var fileName = path[path.length - 1];
+  var pathToFile = "./files/" + fileName + ".json";
+
+if (req.method === "GET" ) { // Read the file requested
```

Comment:
I recommend using express in the future, it makes this ALOT easier. However, you did a good job using a vanilla HTTP server.